src/ctirs/api/v1/stix_files/views.py

```diff
@@ -91,15 +80,12 @@ def get_stix_files(request):
     return JsonResponse(l, safe=False)
 
 
-# STIXファイル追加
 # POST /api/v1/stix_files
 def upload_stix_file(request):
```

Comment:
No decorator.

Reply:
`upload_stix_file` calls from `stix_files` only.
`stix_files` has `@api_key_auth`.
So authentication is complete at this stage.